api/admin.py

```diff
@@ -1,11 +1,23 @@
 from django.contrib import admin
 
 from api.models import Pattern, SquadToken, Submission
-from api.tasks import submit_to_lava
+from api.tasks import submit_to_lava, check_job_status
+
+
+def check_status(modeladmin, request, queryset):
+    for pattern in queryset:
+        check_job_status.delay(pattern, {
+            'job': pattern.lava_job_id,
+            'description': 'something',
+            'status': "Submitted",
+            'pipeline': True,
```

Comment:
I think we need to assume we're only serving v2 jobs and remove all the code for v1. At the same time there should be a check for the test job version before downloading the results. I'm not sure if the 'results.*' api exists for v1 jobs (unlikely). Using 'results.*' would simplify code a lot.